src/Servers/Connections.Abstractions/src/FileHandleEndPoint.cs

```diff
@@ -24,6 +25,7 @@ public FileHandleEndPoint(ulong fileHandle, FileHandleType fileHandleType)
             }
         }
 
+        public bool OwnsHandle { get; private set; }
```

Comment:
```suggestion
        public bool OwnsHandle { get; }
```
I don't see any other set than in the Ctor.